src/main/java/net/fabricmc/example/mixin/ExampleMixin.java

```diff
@@ -1,15 +1,20 @@
 package net.fabricmc.example.mixin;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.TitleScreen;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(TitleScreen.class)
-public class ExampleMixin {
+public class ExampleMixin extends Screen {
```

Comment:
Also since this now extends Screen, this doesn't compile since the Screen's constructor would need to be present.

Also I feel this should be abstract

Reply:
Should make this class abstract as mixin wiki recommends mixin classes to be.